{
 "PRed":[
  {
   "PR":{
    "__lazy_json__":"pr_json/223957769.json"
   },
   "data":{
    "bot_rerun":false,
    "migrator_name":"CompilerRebuild",
    "migrator_version":1,
    "name":"Python 3.7, GCC 7, R 3.5.1, openBLAS 0.3.2"
   },
   "keys":[
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "name"
   ]
  }
 ],
 "bad":{
  "exception":"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))",
  "traceback":[
   "Traceback (most recent call last):",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/connectionpool.py\", line 672, in urlopen",
   "    chunked=chunked,",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/connectionpool.py\", line 421, in _make_request",
   "    six.raise_from(e, None)",
   "  File \"<string>\", line 3, in raise_from",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/connectionpool.py\", line 416, in _make_request",
   "    httplib_response = conn.getresponse()",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/http/client.py\", line 1321, in getresponse",
   "    response.begin()",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/http/client.py\", line 296, in begin",
   "    version, status, reason = self._read_status()",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/http/client.py\", line 265, in _read_status",
   "    raise RemoteDisconnected(\"Remote end closed connection without\"",
   "http.client.RemoteDisconnected: Remote end closed connection without response",
   "",
   "During handling of the above exception, another exception occurred:",
   "",
   "Traceback (most recent call last):",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/adapters.py\", line 449, in send",
   "    timeout=timeout",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/connectionpool.py\", line 720, in urlopen",
   "    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/util/retry.py\", line 400, in increment",
   "    raise six.reraise(type(error), error, _stacktrace)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/packages/six.py\", line 734, in reraise",
   "    raise value.with_traceback(tb)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/connectionpool.py\", line 672, in urlopen",
   "    chunked=chunked,",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/connectionpool.py\", line 421, in _make_request",
   "    six.raise_from(e, None)",
   "  File \"<string>\", line 3, in raise_from",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/urllib3/connectionpool.py\", line 416, in _make_request",
   "    httplib_response = conn.getresponse()",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/http/client.py\", line 1321, in getresponse",
   "    response.begin()",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/http/client.py\", line 296, in begin",
   "    version, status, reason = self._read_status()",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/http/client.py\", line 265, in _read_status",
   "    raise RemoteDisconnected(\"Remote end closed connection without\"",
   "urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))",
   "",
   "During handling of the above exception, another exception occurred:",
   "",
   "Traceback (most recent call last):",
   "  File \"/root/repo/cf-scripts/conda_forge_tick/auto_tick.xsh\", line 750, in main",
   "    dry_run=args.dry_run)",
   "  File \"/root/repo/cf-scripts/conda_forge_tick/auto_tick.xsh\", line 98, in run",
   "    migrate_return = migrator.migrate(recipe_dir, attrs, **kwargs)",
   "  File \"/root/repo/cf-scripts/conda_forge_tick/migrators.xsh\", line 460, in migrate",
   "    if requests.get(rendered['source']['url']).status_code == 200:",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/api.py\", line 75, in get",
   "    return request('get', url, params=params, **kwargs)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/api.py\", line 60, in request",
   "    return session.request(method=method, url=url, **kwargs)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/sessions.py\", line 533, in request",
   "    resp = self.send(prep, **send_kwargs)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/sessions.py\", line 668, in send",
   "    history = [resp for resp in gen] if allow_redirects else []",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/sessions.py\", line 668, in <listcomp>",
   "    history = [resp for resp in gen] if allow_redirects else []",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/sessions.py\", line 247, in resolve_redirects",
   "    **adapter_kwargs",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/sessions.py\", line 646, in send",
   "    r = adapter.send(request, **kwargs)",
   "  File \"/opt/conda/envs/run_env/lib/python3.7/site-packages/requests/adapters.py\", line 498, in send",
   "    raise ConnectionError(err, request=request)",
   "requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))",
   ""
  ]
 },
 "conda-forge.yml":{
  "compiler_stack":"comp7",
  "max_py_ver":"37",
  "max_r_ver":"35"
 },
 "feedstock_name":"cbsyst",
 "hash_type":"sha256",
 "meta_yaml":{
  "about":{
   "home":"https://github.com/oscarbranson/",
   "license":"MIT",
   "license_file":"LICENCE.txt",
   "summary":"Tools for calculating ocean C and B chemistry"
  },
  "build":{
   "noarch":"python",
   "number":"1",
   "script":" -m pip install . --no-deps --ignore-installed -vvv"
  },
  "extra":{
   "recipe-maintainers":[
    "ocefpaf",
    "ocefpaf",
    "ocefpaf"
   ]
  },
  "package":{
   "name":"cbsyst",
   "version":"0.3.5"
  },
  "requirements":{
   "host":[
    "python >=3",
    "pip",
    "python >=3",
    "pip",
    "python >=3",
    "pip"
   ],
   "run":[
    "python >=3",
    "numpy",
    "scipy",
    "uncertainties",
    "tqdm",
    "pandas",
    "python >=3",
    "numpy",
    "scipy",
    "uncertainties",
    "tqdm",
    "pandas",
    "python >=3",
    "numpy",
    "scipy",
    "uncertainties",
    "tqdm",
    "pandas"
   ]
  },
  "source":{
   "sha256":"edd5f9a8c50b9b574698b8111c35f2bc2f6981d67eb352eaf27e39fadb76eefa",
   "url":"https://github.com/oscarbranson/cbsyst/archive/v0.3.5.tar.gz"
  },
  "test":{
   "imports":[
    "cbsyst",
    "cbsyst",
    "cbsyst"
   ]
  }
 },
 "name":"cbsyst",
 "new_version":"0.3.6",
 "pinning_version":"2018.10.16",
 "raw_meta_yaml":"{% set version = \"0.3.5\" %}\n\npackage:\n  name: cbsyst\n  version: {{ version }}\n\nsource:\n  url: https://github.com/oscarbranson/cbsyst/archive/v{{ version }}.tar.gz\n  sha256: edd5f9a8c50b9b574698b8111c35f2bc2f6981d67eb352eaf27e39fadb76eefa\n\nbuild:\n  number: 1\n  noarch: python\n  script: \"{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv\"\n\nrequirements:\n  host:\n    - python >=3\n    - pip\n  run:\n    - python >=3\n    - numpy\n    - scipy\n    - uncertainties\n    - tqdm\n    - pandas\n\ntest:\n  imports:\n    - cbsyst\n\nabout:\n  home: https://github.com/oscarbranson/\n  license: MIT\n  license_file: LICENCE.txt\n  summary: 'Tools for calculating ocean C and B chemistry'\n\nextra:\n  recipe-maintainers:\n    - ocefpaf\n",
 "req":{
  "__set__":true,
  "elements":[
   "numpy",
   "pandas",
   "pip",
   "python",
   "scipy",
   "tqdm",
   "uncertainties"
  ]
 },
 "smithy_version":"3.1.12",
 "time":1568135284.5067306,
 "url":"https://github.com/oscarbranson/cbsyst/archive/v0.3.5.tar.gz",
 "version":"0.3.5"
}